components/scroller/modules/scroller-item.vue

```diff
@@ -0,0 +1,232 @@
+<script>
```

Comment:
Let's put a comment at the top stating that this is code from an external library and link to the library it's from. Also state that we have modified it for our own specific use. Put the same comment in the other library code files.